journal/original/09-chapter-8.adoc

```diff
@@ -267,6 +266,7 @@ where I was with them in settling the affairs of the church.
 Justice Taverner`'s wife came to me, and told me, that the Major General, Colonel Teve,
 and others were below, staying to speak with me; so I went down to them.
 They were courteous, and said they only stayed to see me,
+//Do you want "stayed" above to be switched back to "staid"?
```

Comment:
Nope, we decided to change this word in all editions because its an archaic spelling that is not recognized now. So lets just leave it. In fact, if you would have tried to change it, the linter would have gotten angry!! :)

Reply:
Gotta keep the linter happy!!! lol